tools/scripts/authz/dao_genesis_authorizations.json

```diff
@@ -116,6 +116,78 @@
     },
     "expiration": "2500-01-01T00:00:00Z"
   },
+  {
```

Comment:
Is this necessary? (Given the other PR that went in)

Reply:
No and I don't see it in GitHub changes. Do you?
<img width="320" alt="Screenshot 2024-08-02 at 11 58 14 AM" src="https://github.com/user-attachments/assets/aa9cc46e-8440-4fd2-88da-8af98634f9ba">